v2/api/authorization/extensions/groupversion_info_gen.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
In the feature spec document by @theunrepentantgeek, the package name suggested was `extensions` and it made more sense to have the package being called as `extensions` rather than `customizations`.

Reply:
The subpackage for the interface definitions was spec'd as `genruntime/extensions`, but to avoid name colllisions and other issues, the non-versioned packages were spec'd as `batch/customization`.

So, you're both right. :grin:

Reply:
Have changed the package to be called `customizations`.